tests/virt_autotest/sriov_pci_passthrough.pm

```diff
@@ -0,0 +1,279 @@
+# SUSE's openQA tests
+ #
+ # Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+ 
+# Summary: A test on KVM host to pass a SR-IOV NIC VF to guest via libvirt.
```

Comment:
Will extend for xen, right?

Reply:
Yes. it comes from my initial design when the passthrough was not supported on XEN. Since it is supported on snapshot 12, the xen should have the similar steps with kvm